docs/source/examples/private_utxo/private_utxo_transaction_family.rst

```diff
@@ -0,0 +1,379 @@
+amount of the AssetType and the signers holdings for that AssetType are
+incremented by the transaction amount.
+
+If the AssetType does not exist then the AssetType is create with the Amount
+from the transaction and the signers holdings for that AssetType are set to
+the transaction amount.
+
+In all other cases the transaction fails, for example if the AssetType exists
+but the signer is not the issuer of the AssetType.
+
+
+Transfer Asset
+--------------
+
+Transfer Asset transactions are used to move assets from one participants
+holdings to another participants holdings.
+
+
+Transfer Asset transactions are validated as follows:
+
+The signer of the transaction is calculated.
+
+If the `asset_type` of the transaction does not exist the transaction fails.
+If the signers holdings of the asset type is less then the transaction amount
+the transaction fails.
+
+If the above validation steps pass then the transaction amount is removed from
+the signers holdings and added to the transaction recipients holdings. If the
+recipients holdings do not exists they are created.
+
+
+Convert To UTXO
+---------------
+
+Convert To UTXO transactions are used to move assets from ledger holdings to
+an off ledger UtxoDocument for private trading.
+
+Convert To UTXO transactions are validated as follows:
+
+The signer of the transaction is calculated.
+
+If the `asset_type` of the transaction does not exist the transaction fails.
+If the signers holdings of the asset type is less then the transaction amount
+the transaction fails.
+
+The UtxoDocument is created using the signer as the as the document owner, and
+the asset_type, amount, and nonce from the transaction. The UtxoDocument
+address is computed and and compared to the transactions output_utxo field. If
+the addresses do not match the transaction fails.
+
+If the above validation steps pass then the transaction amount is removed from
+the signers holdings and added to the transaction output_utxo address is added
+to the UTXO store.
+
+
+Convert From UTXO
+-----------------
+
+Convert From UTXO transactions are used to move off ledger assets held in
+UtxoDocuments to on ledger holdings.
+
+Convert From UTXO transactions are validated as follows:
+
+The signer of the transaction is calculated. The transaction document is
+unpacked. The UtxoDocument address of the transaction document is calculated.
+
+If the UtxoDocument address does not exist the transaction fails.
+
+If the `asset_type` of the document does not exist the transaction fails. This
+is done to ensure system consistency.
+
+If the above validation steps pass then the document amount is added to
+the signers holdings and added to the document address is removed
+from the UTXO store.
+
+Transfer UTXO
+-----------------
+
+Transfer UTXO transactions to facilitate off ledger trading.
+
+Transfer UTXO transactions are validated as follows:
+
+The transaction attestation is validated that it is signed by IAS, the Enclave
```

Comment:
The Enclave portion of this sentence looks like it was not properly edited from an earlier revision.  It currently makes no sense.